### PR TITLE
[Incident Report] Add template.

### PR DESCRIPTION
**Pull request by rpavlovs**

### CODE REVIEW

**Comment by Duelist** [comment]
[[Incident] Create a template for future incident reports.](https://trello.com/c/vxTy4UVa/174-incident-create-a-template-for-future-incident-reports)

File: Incident Reports/template.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+
+## Root Cause
+
+* give a detailed explanation of event
+* do not sugarcoat
+
+
+
+## Resolution and recovery
```
Comment (garrett-gottlieb) [inline Incident Reports/template.md]: `Recovery`

File: Incident Reports/template.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+* when staff was notified
+* actions, events, …
+* when service was restored
+
+
+
+## Root Cause
+
+* give a detailed explanation of event
+* do not sugarcoat
```
Comment (garrett-gottlieb) [inline Incident Reports/template.md]: Capitalize first letter of each point

File: Incident Reports/template.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+| Response Time | Recovery Time |
+| :---:         | :---:         |
+| XXd XXh XXm   | XXd XXh XXm   |
+
+
+## Corrective and Preventative Measures
+
+* itemized list of ways to prevent it from happening again
+* what can we do better next time?
```
Comment (garrett-gottlieb) [inline Incident Reports/template.md]: `* Add links to relevant Trello cards`

File: Incident Reports/template.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+
+## Timeline
+
+* list the timezone
+* covers the outage duration
+* when outage began
+* when staff was notified
+* actions, events, …
+* when service was restored
```
Comment (garrett-gottlieb) [inline Incident Reports/template.md]: What if this was a list of times so the reader can copy the format?

File: Incident Reports/template.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+# [Incident Report Title]
+[Date]
+
+
+## Issue Summary
+
+* short summary (5 sentences)
+* list the duration along with start and end times (include timezone)
+* state the impact (most user requests resulted in 500 errors, at peak 100%)
+* close with root cause
```
Comment (garrett-gottlieb) [inline Incident Reports/template.md]: Same here. Format this in sentences so the reader can copy the format.

**Review by garrettg123 — CHANGES_REQUESTED** [review]
👍

File: Incident Reports/template.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+
+Short summary (5 sentences). List the duration along with start and end times (include timezone). State the impact (most user requests resulted in 500 errors, at peak 100%). Close with root cause.
+
+
+
+## Timeline ([Timezone])
+
+* June 13 09:53 PM: Cover the outage duration.
+* June 14 06:02 AM: When outage began
+* June 14 02:08 PM: When staff was notified
```
Comment (garrett-gottlieb) [inline Incident Reports/template.md]: `were`

File: Incident Reports/template.md [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+## Root Cause
+
+* Give a detailed explanation of event
+* Do not sugarcoat
+
+
+
+## Resolution and Recovery
+
+Give detailed explanation of actions taken (includes times)
```
Comment (garrett-gottlieb) [inline Incident Reports/template.md]: `Give a detailed...`

Comment (garrett-gottlieb) [inline Incident Reports/template.md]: `(including times).`

**Review by garrettg123 — APPROVED** [review]
👍

**Pull request merged**